### PR TITLE
[docs]: cloud hil review tool calls

### PR DESCRIPTION
**Pull request by isahers1**

### CODE REVIEW

File: docs/docs/cloud/how-tos/human_in_the_loop_review_tool_calls.md [hunk 1]
```diff
@@ -0,0 +1,575 @@
...
+=== "Python"
+
+    ```python
+    input = {"messages": [{"role": "user", "content": "what's the weather in sf?"}]}
+
+    async for chunk in client.runs.stream(
+        thread["thread_id"],
+        "agent",
+        input=input,
+        stream_mode="values",
```
Comment (vbarda) [inline docs/docs/cloud/how-tos/human_in_the_loop_review_tool_calls.md:233]: i think we need to pass  an interrupt here

Comment (vbarda) [reply docs/docs/cloud/how-tos/human_in_the_loop_review_tool_calls.md:233]: ah it's set on the graph. i think it's worth mentionig up top

Comment (isahers1) [reply docs/docs/cloud/how-tos/human_in_the_loop_review_tool_calls.md:233]: graph was defined as such: `builder.compile(checkpointer=memory, interrupt_before=["human_review_node"])` so I think we are OK? I ran the code and it worked, but idk

Comment (vbarda) [reply docs/docs/cloud/how-tos/human_in_the_loop_review_tool_calls.md:233]: yup!, i think it's just worth mentioning that interrupt is defined in the example that we link to, so people understand where it's coming from

**Review by vbarda — APPROVED** [review]

File: docs/docs/cloud/how-tos/human_in_the_loop_review_tool_calls.md (hunk 1 above)
Comment (isahers1) [reply docs/docs/cloud/how-tos/human_in_the_loop_review_tool_calls.md:233]: ah yeah sorry missed your second comment - will add a mention rn

**Pull request merged**